tests/volatile-arrays.phpt

```diff
@@ -54,7 +56,7 @@ object(Threaded)#%d (%d) {
       }
     }
   }
-  [0]=>
+  ["0"]=>
```

Comment:
@SirSnyder are you sure we want to ignore this instead of correcting it? It is a BC break after all.

Reply:
@dktapps nope, my fault. I assumed this would be the new and intended behavior of php arrays. But it's not, will have a look at it.